en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,245 @@
+
+    // line 12
+    config = merge.recursive(true, config, loadConfig('./vendor/open-orchestra/open-orchestra-media-admin-bundle/OpenOrchestra/GruntTasks/Options/'));
+
+Modify the main javascript task (``grunt_task/javascript_task.js``):
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        grunt.registerTask('javascript',
+            [
+                'coffee:discovering',
+                'coffee',
+                'concat:smartadminjs',
+                'concat:libjs',
+                'concat:orchestrajs',
+                'concat:mediajs',
+                'concat:js'
+            ]
+        );
+    };
+
+Modify the main css task (``grunt_task/css_task.js``):
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        grunt.registerTask('css',
+            [
+                'less:discovering',
+                'less',
+                'concat:smartadmincss',
+                'concat:libcss',
+                'concat:orchestracss',
+                'concat:mediacss',
+                'concat:css',
+                'cssmin'
+            ]
+        );
+    };
+
+In the ``grunt_tasks/options/concat.js.js`` file:
+
+.. code-block:: javascript
+
+    module.exports = {
+        src: [
+            'web/built/smartadmin.js',
+            'web/built/lib.js',
+            'web/built/orchestra.js',
+            'web/built/media.js'
+        ],
+        dest: 'web/js/all.js'
+    };
+
+And in the ``grunt_tasks/options/concat.css.js`` file:
+
+.. code-block:: javascript
+
+    module.exports = {
+        src: [
+            'web/built/smartadmin.css',
+            'web/built/lib.css',
+            'web/built/orchestra.css',
+            'web/built/media.css'
+        ],
+        dest: 'web/css/all.css'
+    };
+
+At that point, you have to run the grunt command (``./node_modules/.bin/grunt``) to create the new js and css files.
+
+Adding your own assets files
+----------------------------
+
+In the following section I will talk only about javascript files, but the process will be the same for the css files.
+
+Adding a single file
+~~~~~~~~~~~~~~~~~~~~
+
+To add the ``bar.js`` file from the ``FooBundle``, the easiest way will be to add the file directly to the
+``grunt_tasks/options/concat.js.js`` file :
+
+.. code-block:: javascript
+
+    module.exports = {
+        src: [
+            'web/built/smartadmin.css',
+            'web/built/lib.css',
+            'web/built/orchestra.css',
+            'web/bundles/foobundle/js/bar.js'
+        ],
+        dest: 'web/css/all.css'
+    };
+
+Adding multiple files
+~~~~~~~~~~~~~~~~~~~~~
+
+To a add multiple file, you could add all of them directly in the ``concat.js.js`` file. This solution will
+result in a gigantic and unmaintainable concatenation file.
+
+A more common way would be :
+
+- create a directory with all your tasks (``grunt_tasks/foo``)
```

Comment:
Are you talking about a bundle the reader is developping ? or is it a downloaded bundle ?
I assume you're talking about the 1st case. If true, is the dir "grunt_tasks/foo" located in the bundle or in the app ?